help/status-and-availability.md

```diff
@@ -75,6 +75,22 @@ and compose box. Status emoji and status messages are also shown on
 
 {end_tabs}
 
+### Configure how a user status is displayed in the right sidebar
+
+You can choose to display both the status emoji and status message, or only the
+status emoji.
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{settings_tab|display-settings}
+
+1. Under **Advanced**, select **Compact** or **Show status and text** for the
+   user list style.
+
```

Comment:
Let's also add a tip that you can hover over the user's name to see a tooltip with the status text.